test/run-k8s-integration.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+
+readonly PKGDIR=${GOPATH}/src/sigs.k8s.io/gcp-filestore-csi-driver
```

Comment:
Why is PKGDIR upcased when all the other local ones are downcased?

Reply:
I don't see any specific reason, the small caps variables are picking values from env variable(set by the caller) or some default value